packages/components/Button/src/Button.types.ts

```diff
@@ -9,9 +9,10 @@ import { IViewProps } from '@fluentui-react-native/adapters';
 
 export const buttonName = 'Button';
 export type ButtonSize = 'small' | 'medium' | 'large';
-export type ButtonAppearance = 'primary' | 'subtle';
+export type ButtonAppearance = 'primary' | 'subtle' | 'ascent' | 'outline';
```

Comment:
>'ascent' [](http://example.com/codeflow?start=54&length=8)

nit: typo? Is this supposed to be 'accent'?

Reply:
Did we finalize whether we want an accent button at all? Also, "accent" not "ascent"?